medicines/api/src/product.rs

```diff
@@ -33,14 +34,14 @@ pub fn handle_doc(document: &IndexResult, products: &mut Vec<Product>) {
     }
 }
 
-pub async fn get_products_by_substance_name(
-    substance_name: &str,
+pub async fn get_substance_with_products(
+    name: &str,
     client: &impl Search,
-) -> Vec<Product> {
+) -> Substance {
     // Get a list of documents from Azure which are about products containing the
```

Comment:
Is this comment displayed somewhere?  if not then it doesn't appear to add anymore information than the function name itself - other than saying Azure

Reply:
I agree, it's pointless if it's not docs